content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5371,18 +5371,12 @@ const getUsers = await prisma.user.findMany({
   where: {
     pets: {
       path: ['sanctuaries'],
-      array_starts_with: ['RSPCA'],
+      array_starts_with: 'RSPCA',
     },
   },
 })
 ```
 
-<Admonition type="info">
-
-**Note**: In PostgresSQL, the value of `array_starts_with` must be an array and not a string, even if the array only contains a single value.
-
-</Admonition>
```

Comment:
Source for change?

Reply:
Internal slack conversation: https://prisma-company.slack.com/archives/C02FZENHD4N/p1680705609602039